dspace-api/src/main/java/org/dspace/app/util/InitializeEntities.java

```diff
@@ -175,7 +189,8 @@ private String getString(String leftCardinalityMin,Element node, String minOrMax
 
     private void populateRelationshipType(Context context, String leftType, String rightType, String leftwardType,
                                           String rightwardType, String leftCardinalityMin, String leftCardinalityMax,
-                                          String rightCardinalityMin, String rightCardinalityMax)
+                                          String rightCardinalityMin, String rightCardinalityMax,
+                                          Boolean copyLeft, Boolean copyRight)
```

Comment:
What does it mean copyLeft and copyRight, What does it do? 
Is CopyLeft the copy record from left to right entity, or copy from right to left? 

Perhaps the naming should include "from" or "to" to indicate the direction of the copy. What do you think?

Reply:
This has been updated